lib/galaxy/config.py

```diff
@@ -383,6 +390,14 @@ def __init__( self, **kwargs ):
         self.galaxy_infrastructure_url = galaxy_infrastructure_url
         self.galaxy_infrastructure_url_set = galaxy_infrastructure_url_set
 
+        # Verify host:port is available before continuing
+        try:
+            q = bind(*re.sub(r'http(s?)\://', '', self.galaxy_infrastructure_url).split(':'))
```

Comment:
I would vastly prefer that, if we can uncover the underlying problem. I posted a question on the original issue - because I cannot recreate locally or I would just patch paste myself.